WPFClient/Communicator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Client;
+using CorEstafette;
+
+namespace WPFClient
+{
+    class Communicator
+    {
+        private readonly HubConnection connection;
+        private readonly List<string> subscribedTopics;
+
+        public Communicator( )
+        {
+            subscribedTopics = new List<string>();
+            connection = new HubConnectionBuilder()
+                .WithUrl("https://localhost:44392/testhub")
+                .Build();
+
+            connection.StartAsync();
```

Comment:
This will need to be awaited for the connection to be started. For that, this can be moved to another method and called independently from ctor, or use _ = Task.Run(async () => { await connection.StartAsync(); }); until we add dependency injection.